data_processing/insee/sirene/geocodage/DAG-sirene-geocodage-etalab.py

```diff
@@ -138,6 +147,7 @@
     national_files_agregation.set_upstream(geocoding)
     prepare_to_rsync.set_upstream(national_files_agregation)
     prepare_to_rsync.set_upstream(split_by_locality)
+    check_stats_coherence.set_upstream(get_geocode_stats)
```

Comment:
Good question. I am a bit afraid of making the DAG fail next month just because of a bug since I was not able to test a full execution of the DAG.
My plan was to let it run as it is on the 1st, monitor it, and if we are happy with the result to push a few changes to make it skip the rsyncs in case of failure next time.
What do you think?

Reply:
Makes sense, we go for that 🤝